retriever/lib/engine_tools.py

```diff
@@ -131,6 +134,17 @@ def json2csv(input_file, output_file=None, header_values=None, encoding=ENCODING
     subprocess.call(['rm', '-r', input_file])
     return output_file
 
+def sqlite2csv(database):
```

Comment:
Follow PEP8: https://www.python.org/dev/peps/pep-0008/

Reply:
Will do @apoorvaeternity :+1:

Reply:
@apoorvaeternity Should I check my changes if following PEP8 using pycodestyle?

Reply:
@apoorvaeternity  Sorry, Didn't know pycodestyle was the replacement for PEP8 in future Python releases. Made the changes in the code according to PEP8.